client/ayon_deadline/plugins/publish/maya/submit_maya_deadline.py

```diff
@@ -115,7 +115,7 @@ class MayaSubmitDeadline(abstract_submit_deadline.AbstractSubmitDeadline,
     targets = ["local"]
     settings_category = "deadline"
 
-    tile_assembler_plugin = "OpenPypeTileAssembler"
+    tile_assembler_plugin = "DraftTileAssembler"
```

Comment:
For context: this was already the default in settings - so it changing here is basically cosmetics